src/usr/local/www/vpn_ipsec_phase1.php

```diff
@@ -944,6 +969,13 @@ function build_eal_list() {
 	$pconfig['splitconn']
 ));
 
+$section->addInput(new Form_Checkbox(
+	'prfselect_enable',
+	'Allow select PRF',
+	'Enable this to manually select pseudo-random function (PRF). In most cases this is not required.',
```

Comment:
Use this instead: `Enable manual Pseudo-Random Function (PRF) selection`

And then add a `setHelp()` with: `Manual PRF selection is typically not required, but can be useful in combination with AEAD Encryption Algorithms such as AES-GCM`.

Reply:
done